alexandria/core/tests/test_views.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+from django.urls import reverse
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_201_CREATED,
+    HTTP_400_BAD_REQUEST,
+    HTTP_403_FORBIDDEN,
+)
+
+from alexandria.core.models import File
+
+
+@pytest.fixture
+def remove_temp_files(settings):
+    yield
+    for i in settings.THUMBNAIL_CACHE_DIR.glob("**/*"):  # pragma: no cover
```

Comment:
Wouldn't it be cleaner to have a `thumbnail_cache` fixture that uses `tempdir` and mocks `settings.THUMBNAIL_CACHE_DIR` for each run?

Reply:
Absolutely! Done.